.travis.yml

```diff
@@ -1,17 +1,19 @@
 language: rust
-sudo: false
 cache: cargo
-before_script:
-- cargo install -f rustfmt
-- export PATH=$PATH:$HOME/.cargo/bin
 rust:
-- nightly
-- beta
-- stable
+  - stable
+  - beta
+  - nightly
+env:
+  - RUN=TEST
+script:
+  - cargo test
 matrix:
+  include:
+    - rust: nightly
```

Comment:
I prefer to use the stable channel for rustfmt now, because that's more... stable ;-)